src/main/java/edu/greenblitz/robotName/subsystems/Gyros/IBaseGyro.java

```diff
@@ -0,0 +1,24 @@
+package edu.greenblitz.robotName.subsystems.Gyros;
+
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public interface IBaseGyro {
```

Comment:
I believe this name is confusing - `I` is used as a prefix for interfaces but `base` is usually reserved for abstract classes

Reply:
abstract interface?

Reply:
Please explain what you mean

I meant that I think the interface's name shouldn't have the word 'Baee' in it

Reply:
i'll just change the name of the class

Reply:
What class? We're talking about an interface

Reply:
i changed the name of the interface